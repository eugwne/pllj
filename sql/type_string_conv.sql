CREATE or replace FUNCTION echo_new_arr(arg text[]) RETURNS text[] AS $$ 
return '{d,e}' 
$$ LANGUAGE pllj;

CREATE or replace FUNCTION echo_arr(arg text[]) RETURNS text[] AS $$ 
return arg
$$ LANGUAGE pllj;

SELECT echo_new_arr(array['a', 'b', 'c']);
SELECT echo_arr(array['a', 'b', 'c']);
