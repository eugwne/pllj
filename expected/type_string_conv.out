CREATE or replace FUNCTION echo_new_arr(arg text[]) RETURNS text[] AS $$ 
return '{d,e}' 
$$ LANGUAGE pllj;
CREATE or replace FUNCTION echo_arr(arg text[]) RETURNS text[] AS $$ 
return arg
$$ LANGUAGE pllj;
SELECT echo_new_arr(array['a', 'b', 'c']);
 echo_new_arr 
--------------
 {d,e}
(1 row)

SELECT echo_arr(array['a', 'b', 'c']);
 echo_arr 
----------
 {a,b,c}
(1 row)

